cmake/modules/SIMDExt.cmake

```diff
@@ -1,66 +1,60 @@
 # detect SIMD extentions
 #
-# ARM_NEON_FLAGS
-#
 # HAVE_ARMV8_CRC
```

Comment:
thanks. The commit does not need to be standalone (i.e work without the other commits) if that makes things easier.